src/cls/fifo/cls_fifo_ops.h

```diff
@@ -149,17 +149,18 @@ WRITE_CLASS_ENCODER(update_meta)
 
 struct init_part
 {
-  std::string tag;
   data_params params;
 
   void encode(ceph::buffer::list& bl) const {
     ENCODE_START(1, 1, bl);
+    std::string tag;
```

Comment:
is this for backward compatibility, so if there is a object written with tag info, while reading and writing back with new code it works fine ?

Reply:
Yes. So I wouldn't have to change the 'on disc' format.